examples/searchMostStarredRepos.js

```diff
@@ -1,17 +1,9 @@
-
-
 const GitHubApi = require('github')
 
-
 const github = new GitHubApi({
     // debug: true
```

Comment:
I think we pass `debug: true` in all the other examples, maybe let's do it here, too?